go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/nspcc-dev/rfc6979
 
-go 1.15
+go 1.20
```

Comment:
BTW, the task tells to update to go 1.19 min. Go 1.20 is better, telling this just in case.